model/ExpandedLicensing/Individuals/NoneLicense.md

```diff
@@ -3,19 +3,18 @@ SPDX-License-Identifier: Community-Spec-1.0
 # NoneLicense
 
 ## Summary
+An Individual Value for License no license is present.
 
-Individual representing an assertion that no license information is present, as applicable.
 
 ## Description
 
-A NoneLicense is the value that is used to indicate
-the absence of license information from a software artifact.
+The SPDX document creator concludes there is no license available for this package
```

Comment:
1. SPDX document creator, or SPDX data creator? I'm assuming "data" creator since I gather for 3.0 this might not be in the context of an SPDX document.
2. I would probably change "package" to "software artifact" since this could also be used for Files or Snippets

Reply:
Updated per suggestions